.github/workflows/post_release_version_bump.yml

```diff
@@ -0,0 +1,39 @@
+name: Automated Release Version Bumps
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  post_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pennylane
+
+      - name: Run version bump
+        run: >
+          python .github/workflows/vb_script.py
+          --version_path "./pennylane_lightning/_version.py"
+          --changelog_path "./.github/CHANGELOG.md" --post_release --post_pl_release
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: post release version bump
+          title: Version Bump
+          body: updated changelog and _version.py
+          branch: post-release-version-bump
+          reviewers: Jaybsoni
```

Comment:
Will change this to whom ever this plugin is assigned to as a reviewer

Reply:
Feel free to add me here, ccing @mlxd too: Jay is adding an automated version bump for the pre and post-release of Lightning. The new actions would tidy the changelog and increment the version number. For now, the action opens a new PR and this field determines who'll be put as the reviewer(s) to the PR. Would you also like to be tagged? :slightly_smiling_face:

Reply:
```suggestion
          reviewers: antalszava
```